docs/docs/addons-shallow-renderer.md

```diff
@@ -0,0 +1,62 @@
+---
+id: shallow-renderer
+title: Shallow Renderer
+permalink: docs/shallow-renderer.html
+layout: docs
+category: Reference
+prev: test-utils.html
+next: animation.html
+---
+
+**Importing**
+
+```javascript
+import ReactShallowRenderer from 'react-test-renderer/shallow'; // ES6
+var ReactShallowRenderer = require('react-test-renderer/shallow'); // ES5 with npm
+```
+### Shallow Rendering
+
+Shallow rendering lets you render a component "one level deep" and assert facts about what its render method returns, without worrying about the behavior of child components, which are not instantiated or rendered. This does not require a DOM.
+
+ - [`shallowRenderer.render()`](#shallowrenderer.render)
```

Comment:
The pattern we show first is
```
const ReactShallowRenderer = require('react-test-renderer/shallow');
const renderer = new ReactShallowRenderer();
```
It makes sense to reserve the UpperCamelCase for the class itself, and for instances of the class use lowerSnakeCase. What we need still is consistency between the naming of the instances. I would vote for `shallowRenderer` instead of `renderer`.